packages/lerna/src/commands/add-caching/index.ts

```diff
@@ -55,16 +64,23 @@ class AddCachingCommand extends Command {
 
     log.pause();
 
+    const existingTargetDefaults = this.uniqueScriptNames.filter(
+      (scriptName) => nxJson.targetDefaults?.[scriptName]?.dependsOn?.length
+    );
     const { targetDefaults } = await inquirer.prompt<{ targetDefaults: UserAnswers["targetDefaults"] }>([
       {
         type: "checkbox",
         name: "targetDefaults",
         message:
           "Which scripts need to be run in order? (e.g. before building a project, dependent projects must be built.)\n",
         choices: this.uniqueScriptNames,
+        default: existingTargetDefaults,
```

Comment:
This allows inquirer to pre-select all targets that already have dependsOn configured

<img width="785" alt="image" src="https://github.com/lerna/lerna/assets/6913035/59afde67-9522-4a4d-9ba3-c9419aa85273">